test/configureLogger.js

```diff
@@ -6,6 +6,7 @@ const snowflake = require('../lib/snowflake');
  * @param logLevel one of OFF | ERROR | WARN | INFO | DEBUG | TRACE
  */
 exports.configureLogger = (logLevel = 'ERROR') => {
-  Logger.setInstance(new NodeLogger());
+  Logger.getInstance().closeTransports();
```

Comment:
it would be better first to set the new one an then close the old one to be sure not to lose any logs

Reply:
This method is used only for the tests so IMO it is fine with that approach - no need to worry for the logs to be lost.